package.json

```diff
@@ -34,6 +34,7 @@
     "c8": "^9.0.0",
     "execa": "^8.0.1",
     "find-up": "^7.0.0",
-    "glob": "^10.3.10"
+    "glob": "^10.3.10",
+    "tsc-alias": "^1.8.8"
```

Comment:
As I said in https://github.com/mcollina/borp/issues/21#issuecomment-2003398097, we should not add this dependency.

Reply:
Thanks @mcollina,  probably I undervalued the comment. 

`tsc-alias` could be moved  into the  **devDependency**, but I'm not sure it is what you mean.

Alternatively, we could allow the user via a parameter to define certain functions to be launched post-compilation of typescript.
Something like `borp --coverage --post-compile=postCompile.ts`
In this way, borp would not be depends to tsc-alias at all, but it would be the user's responsibility to decide what to do in the postCompile.ts file.

Reply:
The latter would be great.

Reply:
Ok thanks, I will work on it.

Reply:
Hello @mcollina , 
I'm ready to push the new code, do you prefer a new PR or can I use the same one, eventually rewriting the git history?

Reply:
keep this